gradle.properties

```diff
@@ -7,6 +7,6 @@ org.gradle.jvmargs=-Xmx1G
 	plasma_build=18
 
 # Mod Properties
-	mod_version = 1.1.0
-	maven_group = io.github.minecraftcursedlegacy
+	mod_version = 1.1.1
+	maven_group = io.github.minecraft-cursed-legacy
```

Comment:
our package doesn't have the dashes

Reply:
therefore our maven group doesn't

Reply:
yes I know our website has them but I'm not refactoring every project on this org just for that

Reply:
also this is what we're under on half's maven

Reply:
> yes I know our website has them but I'm not refactoring every project on this org just for that

I don't expect you to. Dashes are invalid for package names.
But the Maven group *should* match the domain name.

The reason I changed it is because when I switched from the online copy to my local copy, the dependency wasn't found until I changed it accordingly.

Reply:
And it already has dashes in the example mod.

https://github.com/minecraft-cursed-legacy/Example-Mod/blob/b1.7.3/build.gradle#L55

Half's maven isn't browseable, so all I know is MCL isn't mentioned in the unformatted XML file it links to, and I can't resolve it as a dependency using that repo, so it must not be in there at all.